package.json

```diff
@@ -37,7 +37,7 @@
   },
   "dependencies": {
     "@fontsource/jetbrains-mono": "^4.5.11",
-    "@nasa-jpl/stellar": "^1.0.14",
+    "@nasa-jpl/stellar": "^1.0.18",
```

Comment:
We can remove updates to `package.json` and `package-lock.json` since https://github.com/NASA-AMMOS/aerie-ui/pull/204